### PR TITLE
From elisoncrom fixes misspellings

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:**  

contributions from @elisoncrum

Addresses no open issue.

Fixes 70 major and minor spelling errors within 54 files either .md, .js, or .ts files.

## Changes
- fixed several minor misspellings in current and past .md files with misspellings
- fixed possibly major misspelling within `packages/integration-tests/api/executions.js`
  - Line 46 incorrectly references `@cumulus/integration-tests/exeuctions.getExecutionStatus` which does not exist. Changes `exeuctions` to `executions`
- fixed test output and documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1661,7 +1661,7 @@ new `update-granules-cmr-metadata-file-links` task.
   - Update reports to return breakdown by Granule of files both in DynamoDB and S3
 - **CUMULUS-2123**
   - Added `cumulus-rds-tf` DB cluster module to `tf-modules` that adds a
-    severless RDS Aurora/ PostgreSQL  database cluster to meet the PostgreSQL
+    serverless RDS Aurora/ PostgreSQL  database cluster to meet the PostgreSQL
```
Comment (botanical) [inline CHANGELOG.md:1664]:
```suggestion
    serverless RDS Aurora/PostgreSQL database cluster to meet the PostgreSQL
```

File: website/versioned_docs/version-1.11.0/data-cookbooks/setup.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ original_id: setup
 
 # Setup
 
-### Getting setup to work with data-cookboooks
```
Comment (botanical) [inline website/versioned_docs/version-1.11.0/data-cookbooks/setup.md:10]: 👻

File: docs/data-cookbooks/queue-post-to-cmr.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ title: Queue PostToCmr
 hide_title: false
 ---
 
-In this document, we walktrough handling CMR errors in workflows by queueing PostToCmr. We assume that the user already has an ingest workflow setup.
+In this document, we walkthrough handling CMR errors in workflows by queueing PostToCmr. We assume that the user already has an ingest workflow setup.
```
[the remark this answers is not in the document]
Comment (flamingbear) [reply docs/data-cookbooks/queue-post-to-cmr.md:7]: https://www.englishforums.com/English/WalkthroughWalkThroughWalk-Through/bgbhmd/post.htm
 
Yeah in this case it's a verb and should be two words.

File: website/versioned_docs/version-v1.14.2/data-cookbooks/sns.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ SfSnsReport:
...
+Additional listeners to the SF tracker topic can be configured in `app/config.yml` under `sns.sftracker.subscriptions`. Shown below is configuration that subscribes an additional lambda function (`SnsS3Test`) to receive broadcasts from the `sftracker` SNS. The `endpoint` value depends on the protocol, and for a  lambda function, requires the function's Arn. In the configuration it is populated by finding the lambda's Arn attribute via [Fn::GetAtt](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html). Note the lambda name configured in `lambdas.yml` `SnsS3Test` needs to have it's name postpended with `LambdaFunction` to have the Arn correctly found.
```
Comment (botanical) [inline website/versioned_docs/version-v1.14.2/data-cookbooks/sns.md:127]:
```suggestion
Additional listeners to the SF tracker topic can be configured in `app/config.yml` under `sns.sftracker.subscriptions`. Shown below is configuration that subscribes an additional lambda function (`SnsS3Test`) to receive broadcasts from the `sftracker` SNS. The `endpoint` value depends on the protocol, and for a  lambda function, requires the function's Arn. In the configuration it is populated by finding the lambda's ARN attribute via [Fn::GetAtt](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html). Note the lambda name configured in `lambdas.yml` `SnsS3Test` needs to have it's name postpended with `LambdaFunction` to have the Arn correctly found.
```

while we are here, should we update `Arn` to `ARN`? There's a couple other files that have this. Definitely a nit though

File: website/versioned_docs/version-v8.0.0/data-cookbooks/queue-post-to-cmr.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ hide_title: false
 original_id: queue-post-to-cmr
 ---
 
-In this document, we walktrough handling CMR errors in workflows by queueing PostToCmr. We assume that the user already has an ingest workflow setup.
+In this document, we walkthrough handling CMR errors in workflows by queueing PostToCmr. We assume that the user already has an ingest workflow setup.
```
Comment (botanical) [inline website/versioned_docs/version-v8.0.0/data-cookbooks/queue-post-to-cmr.md:8]:
```suggestion
In this document, we walk through handling CMR errors in workflows by queueing PostToCmr. We assume that the user already has an ingest workflow setup.
```

File: website/versioned_docs/version-v8.1.0/data-cookbooks/queue-post-to-cmr.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ hide_title: false
 original_id: queue-post-to-cmr
 ---
 
-In this document, we walktrough handling CMR errors in workflows by queueing PostToCmr. We assume that the user already has an ingest workflow setup.
+In this document, we walkthrough handling CMR errors in workflows by queueing PostToCmr. We assume that the user already has an ingest workflow setup.
```
Comment (botanical) [inline website/versioned_docs/version-v8.1.0/data-cookbooks/queue-post-to-cmr.md:8]:
```suggestion
In this document, we walk through handling CMR errors in workflows by queueing PostToCmr. We assume that the user already has an ingest workflow setup.
```

**Review by botanical — APPROVED** [review]
a couple nits, otherwise thanks so much for doing this! Had a laugh at all the cookboooks 👻

File: docs/data-cookbooks/queue-post-to-cmr.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (flamingbear) [reply docs/data-cookbooks/queue-post-to-cmr.md]: 7ca35de3a4  fixes all of them that I could find

File: website/versioned_docs/version-v1.14.2/data-cookbooks/sns.md (hunk 4 above)
Comment (flamingbear) [reply website/versioned_docs/version-v1.14.2/data-cookbooks/sns.md]: You're right.  Nit but since we're here and you saw it 02b0d6a7b3

**Pull request merged**